src/main/java/org/mule/module/apikit/validation/body/form/transformation/MultipartFormData.java

```diff
@@ -96,11 +99,27 @@ public void addDefault(String key,String value){
     multipartEntityBuilder.addTextBody(key,value);
   }
 
-  public InputStream build() throws InvalidFormParameterException{
-    try {
-      return multipartEntityBuilder.build().getContent();
-    } catch (IOException e) {
-      throw new InvalidFormParameterException(e);
+  public MultipartFormData build(){
+    this.httpEntity = multipartEntityBuilder.build();
+    return this;
+  }
+
+  public InputStream getContent() throws InvalidFormParameterException{
+    if (httpEntity == null || httpEntity.getContentLength() < 0L) {
+      throw new InvalidFormParameterException("Content length is unknown");
```

Comment:
Problem when resolving form parameters, content lenght is unknow blabla.. explain the exact problem

Reply:
Validation wasn't really necessary, since as we are adding each part the library takes care of the content length